internal/testrunner/runners/pipeline/runner.go

```diff
@@ -311,6 +311,11 @@ func verifyDynamicFields(result *testResult, config *testConfig) error {
 func verifyFieldsInTestResult(result *testResult, fieldsValidator *fields.Validator) error {
 	var multiErr multierror.Error
 	for _, event := range result.events {
+		err := checkErrorMessage(event)
+		if err != nil {
+			multiErr = append(multiErr, err)
```

Comment:
Should we `continue` here? IOW, should we check fields in the event if we find the `error.message` field?

Reply:
Hmm... maybe you're right. Actually all other fields can be wrong, e.g. should be removed but weren't due to an error.

Reply:
Fixed